index.html

```diff
@@ -1,247 +1,380 @@
-    <section id="about">
-      <div class="about">
-        <h2>About me</h2>
-        <p>
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <button>Get My Resume</button>
-      </div>
-      <hr />
-      <div class="skillset">
-        <div class="specifics">
-          <h3><span>&#9830;</span>Language</h3>
-          <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-          </ul>
-        </div>
-        <div class="specifics">
-          <h3><span>&#9744;</span>Frameworks</h3>
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSpec</li>
-            <li>CapyBara</li>
-            <li>Salenium</li>
-          </ul>
-        </div>
-        <div class="specifics">
-          <h3><span>&#9679;</span>Skills</h3>
-          <ul>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
-        </div>
-      </div>
-    </section>
-    <section id="contact">
-      <div class="contact-text">
-        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-      </div>
-      <form action="https://formspree.io/f/xpznwejk" class="form" method="post">
-        <fieldset>
-          <div class="allNames">
-            <div class="input-group allNames1">
-            <label>
-            <input type="text" id="firstName" class="fName" name="firstName" placeholder="First name" required> 
-          </label>
-        </div>
-          <div class="input-group allNames1">
-            <label>
-            <input type="text" id="lastName" class="lName" name="lastName" placeholder="Last name" required>
-          </label>
-          </div>
-          </div>
-          <div class="input-group fullN">
-            <label>
-            <input type="text" id="fullName" name="fullName" placeholder="Full name" required>
-          </label>
-          </div>
-          <div class="input-group">
-            <label>
-            <input type="email" id="email" name="email" placeholder="name@gmail.com" required>
-          </label>
-          </div>
-          <div class="input-group">
-            <label>
-            <textarea name="msg" id="msg" cols="30" rows="10" minlength="30" placeholder="Enter text here" required></textarea>
-          </label>
-          </div>
-          <div class="submit">
-            <button type='submit'>Get in touch</button>
-          </div>
-        </fieldset>
-      </form>
-     </section>
-    <script src="script.js"></script>
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+  <head>

```

Comment:
- [ ] Dear @prg-04, Great work on the required changes, I would love to make suggestions on the changes you made. I see you have replaced the content of your `html` and `css` files. This causes another issue as you can see from the screenshot I have attached below, I cannot see the background images or even the styles. I have used a local and a live version to review your project. Please do the same and make sure :
> 1. The images and styles are back in place
> 2. The comment from the previous reviewer is addressed.

Note: to get back to where you were before this commit you can [rebase ](https://git-scm.com/docs/git-rebase)or [reset ](https://git-scm.com/docs/git-reset) your branch head. 

![Capture](https://user-images.githubusercontent.com/69077061/200112774-6cec9382-5828-41b9-8dd5-239cd3ff11b6.PNG)

Reply:
@Tadesse-Alemayehu i have adjusted the code although it was still working before i made the changes